app/controllers/passwords_controller.rb

```diff
@@ -0,0 +1,21 @@
+class PasswordsController < ApplicationController
```

Comment:
Why an independent password controller, this should be only for users. Also devise must have some functionalities already built in for this purpose. I think you should inherit that, check how you can use devise here, what is the better approach for doing this.

Reply:
devise is to be used in phase3 of the the project
Phase2 needs a custom multiple user authentication, thus this is used.